src/scss/_base.scss

```diff
@@ -22,8 +22,8 @@
 
```

Comment:
In the grid, font size of a focused cell is smaller than the rest of the grid. Should stay at the same size.